.github/workflows/lint-ecosystem-order.yml

```diff
@@ -1,12 +1,6 @@
 name: Lint Ecosystem Order
 
 on:
-  push:
```

Comment:
Whi this has been changed?

Reply:
It wasn't clear on the intent, as the previous version should run on every branch `push` **not** named `main` / `master` which would mean it will run outside of a PR and 2x on a PR (`push` + `pull_request`). In the current version it only runs on PRs (which I believe is the desired state).

In addition, the script will only work on PRs. If we want to support PRs and not PRs, I need to tweak the script to run differently depending on the context & we should probably have 2 workflows.

I also wouldn't understand if its not running on main/master why we'd ever believe its not running on a PR....

Reply:
@jsumners hey just looked at git and says you wrote it, what was intent behind `push` and `pull_request` trigger?

Reply:
I think in past, che `pull_request` event was not triggered when new commits where `push`ed to the PR's branch.

Thanks for the clarification

I think it is good with this lighter setup

cc/ @Fdawgs